airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/streams/base_streams.py

```diff
@@ -40,6 +41,10 @@ class FBMarketingStream(Stream, ABC):
     # entity prefix for `include_deleted` filter, it usually matches singular version of stream name
     entity_prefix = None
 
+    @property
+    def availability_strategy(self) -> Optional["AvailabilityStrategy"]:
```

Comment:
Hey @erohmensing !

Just reviewing for my own understanding

So the gist of this is
1. We have a new default strategy for HttpStream
2. This PR explicitly sets the `availability_strategy` to the old default of `HttpStream` for subclasses of `HttpStream`.
3. This is so that when we release the new default strategy (in another PR) we dont accidentally introduce a regression

Is that all correct?